irmin-website.opam

```diff
@@ -18,6 +18,7 @@ depends: [
   "dune"       {build & >= "1.7.0"}
   "irmin"
   "irmin-unix"
+  "digestif.ocaml"
```

Comment:
this won't work, the package is `digestif` -- `digestif.ocaml` should be added to the `dune` file (in the `libraries` field of the `executable stanza`)

Reply:
I think this is already done?

https://github.com/tarides/irmin.io/pull/7/files#diff-6ca3c6684ec06eee8be2c61772e7f672R13

But it seemingly wasn't sufficient.

Reply:
Ah, OK. I did it for the `ocaml-mdx` stuff, but not for the example executable, as you suggest. Thanks!